source/robot_model/test/tests/test_collisions.cpp

```diff
@@ -0,0 +1,105 @@
+#include "robot_model/Model.hpp"
+
+#include <stdexcept>
+#include <memory>
+#include <gtest/gtest.h>
+
+using namespace robot_model;
+
+class RobotModelCollisionTesting : public testing::Test {
+protected:
+  void SetUp() override {
+    std::vector<std::string> package_paths = {std::string(TEST_FIXTURES) + "ur5e"}; 
```

Comment:
Nitpick but you are mixing naming styles in this file. Sometimes lower_camel_case and sometimes ur5eWithGeometries. Please stick to lower_camel_case